protoutil/conversion_test.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package protoutil provides mechanisms for interacting with proto.
+package protoutil
+
+import (
+	"testing"
+
+	"github.com/abcxyz/pkg/testutil"
+	"github.com/google/go-cmp/cmp"
+	"google.golang.org/protobuf/testing/protocmp"
+	"google.golang.org/protobuf/types/known/structpb"
+)
+
+func TestParseProject(t *testing.T) {
+	t.Parallel()
+
+	cases := []struct {
+		name          string
+		input         any
+		want          *structpb.Struct
+		wantErrSubstr string
+	}{
+		{
+			name: "success",
```

Comment:
To have a failure case, consider: https://stackoverflow.com/questions/33903552/what-input-will-cause-golangs-json-marshal-to-return-an-error

Reply:
done